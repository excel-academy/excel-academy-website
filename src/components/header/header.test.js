import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from './header';

describe('Header', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Header siteTitle="Hello world" />);
    const result = renderer.getRenderOutput();
    // const tree = renderer.create(<Header siteTitle="Hello world" />).toJSON();
    expect(result).toMatchSnapshot();
    // expect(result).toHaveStyleRule('background', 'rebeccapurple');
  });
});
