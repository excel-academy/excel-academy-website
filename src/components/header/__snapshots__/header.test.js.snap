// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<ForwardRef>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "1.45rem 1.0875rem",
      }
    }
  >
    <h1
      style={
        Object {
          "margin": 0,
        }
      }
    >
      <mockConstructor
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
        to="/"
      >
        Hello world
      </mockConstructor>
    </h1>
    <MenuWithQuery />
  </div>
</ForwardRef>
`;
